### PR TITLE
LSR-948 Use Uris instead of paths to support newer SDKs

### PR DESCRIPTION
**Pull request by nhpatt**

### CODE REVIEW

File: android/library/src/main/java/com/liferay/mobile/screens/base/MediaStoreEvent.java [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
 
+import android.net.Uri;
+
 /**
  * @author Víctor Galán Grande
  */
 public class MediaStoreEvent {
 
-	private final String filePath;
+	private final Uri filePath;
```
Comment (dgarciasarai) [inline android/library/src/main/java/com/liferay/mobile/screens/base/MediaStoreEvent.java]: Rename to fileUri?

File: android/library/src/main/java/com/liferay/mobile/screens/base/MediaStoreRequestShadowActivity.java [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
 	public static final String MEDIA_STORE_TYPE = "MEDIA_STORE_TYPE";
 	private int mediaStoreType;
-	private String filePath;
+	private Uri filePath;
```
Comment (dgarciasarai) [inline android/library/src/main/java/com/liferay/mobile/screens/base/MediaStoreRequestShadowActivity.java]: Rename to fileUri?

File: android/library/src/main/java/com/liferay/mobile/screens/base/MediaStoreEvent.java (hunk 1 above)
Comment (nhpatt) [inline android/library/src/main/java/com/liferay/mobile/screens/base/MediaStoreEvent.java]: Changed!

File: android/library/src/main/java/com/liferay/mobile/screens/base/MediaStoreRequestShadowActivity.java (hunk 2 above)
Comment (nhpatt) [inline android/library/src/main/java/com/liferay/mobile/screens/base/MediaStoreRequestShadowActivity.java]: Changed!

File: android/library/src/main/java/com/liferay/mobile/screens/base/MediaStoreRequestShadowActivity.java [hunk 3]
```diff
@@ -43,17 +43,8 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
...
-
-		String path = "";
-
-		if (mediaStoreType == SELECT_IMAGE_FROM_GALLERY) {
-			path = FileUtil.getPath(this, intentData.getData());
-		} else if (mediaStoreType == TAKE_PICTURE_WITH_CAMERA) {
-			path = filePath;
-		}
-
+		Uri path = mediaStoreType == SELECT_IMAGE_FROM_GALLERY ? intentData.getData() : fileUri;
```
Comment (victorg1991) [inline android/library/src/main/java/com/liferay/mobile/screens/base/MediaStoreRequestShadowActivity.java]: Uri uri?

Comment (nhpatt) [inline android/library/src/main/java/com/liferay/mobile/screens/base/MediaStoreRequestShadowActivity.java]: Changed!

**Pull request merged**